include/dts-functions.sh

```diff
@@ -1639,9 +1532,9 @@ show_footer(){
     echo -ne "${RED}${SSH_OPT_UP}${NORMAL} to launch SSH server  ${NORMAL}"
   fi
   if [ "${SEND_LOGS_ACTIVE}" == "true" ]; then
-    echo -ne "${RED}${SEND_LOGS_OPT}${NORMAL} to disable sending DTS logs ${NORMAL}"
+    echo -e "${RED}${SEND_LOGS_OPT}${NORMAL} to disable sending DTS logs ${NORMAL}"
   else
-    echo -ne "${RED}${SEND_LOGS_OPT}${NORMAL} to enable sending DTS logs ${NORMAL}"
+    echo -e "${RED}${SEND_LOGS_OPT}${NORMAL} to enable sending DTS logs ${NORMAL}"
   fi
   if [ "${VERBOSE_ACTIVE}" == "true" ]; then
     echo -ne "${RED}${VERBOSE_OPT}${NORMAL} to disable verbose mode ${NORMAL}"
```

Comment:
Shouldn't you remove `-n` from this echo and those below if you removed it from those above?

Reply:
Check message of the commit d9e30934f2c2e98362eb891a53bbe13bb508d8e4.